src/web/app/pages-instructor/instructor-audit-logs-page/instructor-audit-logs-page.component.ts

```diff
@@ -150,9 +150,11 @@ export class InstructorAuditLogsPageComponent implements OnInit {
                 })),
             mergeAll(),
             finalize(() => this.isLoading = false))
-        .subscribe(((student: Students) =>
-                // Student with no name is selectable to search for all students since the field is optional
-                this.courseToStudents[student.students[0].courseId] = [emptyStudent, ...student.students]),
+        .subscribe(((student: Students) => {
+          student.students.sort((a: Student, b: Student): number => a.name.localeCompare(b.name));
+          // Student with no name is selectable to search for all students since the field is optional
+          this.courseToStudents[student.students[0].courseId] = [emptyStudent, ...student.students];
+        }),
```

Comment:
@jianhandev yes, I removed the TableComparator and replaced with this simple sort. I also updated the snapshot to before any changes were made.